apple/internal/apple_framework_import.bzl

```diff
@@ -434,6 +435,7 @@ def _apple_static_framework_import_impl(ctx):
             label = label,
             libraries = framework.binary_imports,
             linkopts = linkopts,
+            swiftinterface_imports = framework.swift_interface_imports,
```

Comment:
This wasn't needed by tests, but I'm not sure if we fully test the path that would need it.